appveyor.yml

```diff
@@ -32,7 +32,7 @@ init:
 before_build:
   - ps: |
       # Get sources
-      $src = get-childitem -Path *.cpp -Recurse -Exclude pyffish.cpp | select -ExpandProperty FullName
+      $src = get-childitem -Path *.cpp -Recurse -Exclude pyffish.cpp ffish.cp | select -ExpandProperty FullName
```

Comment:
There are typos in `ffishjs.cpp`. And to my knowledge it requires the patterns as comma-separated, that is where the cryptic CI error message might come from.